RoadMap_SwiftUI/RoadMap_SwiftUI/View/MainTabView.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  MainTabView.swift
+//  RoadMap_SwiftUI
+//
+//  Created by Артур Щукин on 26.01.2023.
+//
```

Comment:
```suggestion
//
//  MainTabView.swift
//  RoadMap_SwiftUI
//
//  Created by Артур Щукин on 26.01.2023.
//
```
не отработал форматер